app.js

```diff
@@ -303,3 +303,20 @@ for (let i = 0; i < projects.length; i += 1) {
     });
   });
 }
+
+// create form sumit
+
+const mail = document.getElementById('form-email');
+const form = document.querySelector('.form-media');
+const error = document.querySelector('.errormsg');
+
+form.addEventListener('submit', (event) => {
+  const formEmail = mail.value;
+  if (formEmail.toLowerCase() !== formEmail) {
+    error.innerHTML = 'E-mail address must be lowercase. The form will not be sent <i class="fa-solid fa-triangle-exclamation"></i>';
+    error.classList.add('active');
+    event.preventDefault();
+  } else {
+    error.classList.remove('active');
+  }
```

Comment:
- [ ] [REQUIRED] Please make changes!
- If a user submits the contact form with the `email` field containing `uppercase` characters an error message should show up. Good job so far! You have used the right path, but your validation is not working.
- [HINT] if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.
- [ ] [OPTIONAL] It would be great if you set a validation that only shows **words** not numbers in the `Name` input

![image](https://user-images.githubusercontent.com/63915024/175752836-c99aadab-20ff-4d7b-8582-4b8a4229feac.png)